blockchain/types/transaction.go

```diff
@@ -612,7 +620,7 @@ func (tx *Transaction) AsMessageWithAccountKeyPicker(s Signer, picker AccountKey
 	}
 
 	tx.mu.Lock()
-	tx.validatedIntrinsicGas = &ValidatedIntrinsicGas{Gas: intrinsicGas + gasFrom + gasFeePayer, Tokens: dataTokens}
+	tx.validatedGas = &ValidatedGas{IntrinsicGas: intrinsicGas, SigValidateGas: gasFrom + gasFeePayer, Tokens: dataTokens}
```

Comment:
How about the below structure to maintain the sematic of `IntrinsicGas`?

```suggestion
        sigValidateGas := gasFrom + gasFeePayer
	tx.validatedGas = &ValidatedGas{IntrinsicGas: intrinsicGas + sigValidateGas, SigValidateGas: sigValidateGas, Tokens: dataTokens}
```

Reply:
Done, thanks! f5b3768